module/builder/consensus/builder.go

```diff
@@ -187,9 +188,16 @@ func (b *Builder) BuildOn(parentID flow.Identifier, setter func(*flow.Header) er
 	// 3) If it was already included on the pending part of the chain, skip, but
 	// keep in memory pool for now.
 	// 4) Otherwise, this guarantee can be included in the payload.
-
+	var guaranteeCount uint
```

Comment:
We don't need the extra variable here -- we can just use `len(guarantees)`

Reply:
right, applied at cfb71e4